src/BTVNanoCommissioning/workflows/ctag_Wctt_valid_sf.py

```diff
@@ -96,6 +100,8 @@ def process_shift(self, events, shift_name):
             "cutbased_WcE": "ectag_cutbased_Wc_sf",
             "semittM": "ctag_Wc_sf",  # same histogram representation as W+c
             "semittE": "ectag_Wc_sf",  # same histogram representation as W+c
+            "WcM_noMuVeto": "ctag_Wc_sf",
+            "semittM_noMuVeto": "ctag_Wc_sf",
```

Comment:
Is the semileptonM selection using Wc on purpose?

Reply:
Yes. The only difference between the two is in the nJet distribution and I think the nJet histogram covers a large range already.

Reply:
Sure, pushed updated comments